silkrpc/ethbackend/protocolVersion_client.hpp

```diff
@@ -0,0 +1,87 @@
+/*
```

Comment:
Rename file as `protocol_version_client.hpp` for uniform file names.

Reply:
ok

Reply:
The file `protocolVersion_client.hpp` is still present in the PR

Reply:
ok